webapp.py

```
@@ -0,0 +1,240 @@
+def create_asgi_app():
+    # Expose ASGI wrapper for uvicorn
+    return WsgiToAsgi(create_app())
+
+
+if __name__ == "__main__":
+    # Optional: run with uvicorn if available; otherwise, fallback to Flask dev server
+    try:
+        import uvicorn
+
+        uvicorn.run("webapp:create_asgi_app", host="127.0.0.1", port=8200, factory=True)
+    except Exception:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)